Valet/VALSecureEnclaveValet.m

```diff
@@ -223,9 +223,20 @@ - (nullable instancetype)initWithSharedAccessGroupIdentifier:(nonnull NSString *
     self = [super initWithSharedAccessGroupIdentifier:sharedAccessGroupIdentifier accessibility:accessibility];
     if (self != nil) {
         SEL const backwardsCompatibleInitializer = @selector(initWithSharedAccessGroupIdentifier:accessibility:);
+#if TARGET_IPHONE_SIMULATOR
```

Comment:
Fair. Today `TARGET_IPHONE_SIMULATOR` is `#define`d to be `TARGET_OS_SIMULATOR`... but certainly can't hurt to check both.